README.md

```diff
@@ -52,28 +50,38 @@ Tested with the following:
 
 * `letsencrypt_domain` - Domain the certificate is for.
 * `letsencrypt_email` - Your email as certificate owner.
+* `letsencrypt_tos_sha256` - Provide the SHA256 hash of the latest Terms-of-Service you agreed to
 
 #### Optional
 
-* `letsencrypt_certbot_args` - Additional command line args to Certbot.
-* `letsencrypt_certbot_verbose` - Make Certbot output to console (default `true`).
-* `letsencrypt_certbot_version` - Set specific Certbot version, for example a git tag or branch. Note that the lowest version of Certbot we support is 0.6.0.
+* `letsencrypt_args` - Additional command line args to simp_le LetsEncrypt client.
 * `letsencrypt_force_renew` - Whether to attempt renewal always, default to `true`.
 * `letsencrypt_pause_services` - List of services to stop/start while calling Certbot.
 * `letsencrypt_request_www` - Request `www.` automatically (default `true`).
 
+### Terms of Service
+
+On the [Let's Encrypt: Policy and Legal Repository](https://letsencrypt.org/repository/) page the latest "Terms of Service" PDF document can be downloaded. Make sure you agree to this document before you calculate the checksum and provide it as role input variable.
+
+```bash
+export TOS_DOCUMENT=LE-SA-v1.1.1-August-1-2016.pdf
+wget https://letsencrypt.org/documents/$TOS_DOCUMENT
+shasum -a 256 $TOS_DOCUMENT
+```
+
+At writing time the latest tos_sha256 is `6373439b9f29d67a5cd4d18cbc7f264809342dbf21cb2ba2fc7588df987a6221`
+
 ### Example Playbook
 
 This role works best when included just before your main site role, for example. Or it can be used in an individual playbook, for example as below.
 
-This role should become root on the target host.
+If not manually specified the role will create system user and group "letsencrypt" that executes the client
 
     ---
     - hosts: myhost
-      become: yes
-      become_user: root
       roles:
         - role: ansible-letsencrypt
+          letsencrypt_user: "le-client-user"
```

Comment:
The intention was to show how to override. But it was more confusing than explaining.

done